docs/API.md

```diff
@@ -433,55 +428,6 @@ class App extends React.Component {
 }
 ```
 
-### Named Components
```

Comment:
I think it'd make more sense to keep this example here and possibly just link to here from the `components` section on `<Route>` if we really want to eliminate duplication.

That said I think it's okay to have it in both places though.